app/src/main/java/mod/hey/studios/build/BuildSettingsDialog.java

```diff
@@ -50,10 +50,11 @@ public void show() {
                 addInputPref(BuildSettings.SETTING_ANDROID_JAR_PATH, "", "Custom android.jar", EditorInfo.TYPE_CLASS_TEXT, contentView),
                 addInputPref(BuildSettings.SETTING_CLASSPATH, "", "Classpath (separated by :)", EditorInfo.TYPE_CLASS_TEXT, contentView),
                 addSingleChoicePref(BuildSettings.SETTING_DEXER, new String[]{"Dx", "D8"}, "Dx", "Dexer", contentView),
-                addSingleChoicePref(BuildSettings.SETTING_JAVA_VERSION, BuildSettingsDialogBridge.getAvailableJavaVersions(), "1.8", "Java version", contentView),
+                addSingleChoicePref(BuildSettings.SETTING_JAVA_VERSION, BuildSettingsDialogBridge.getAvailableJavaVersions(), "1.7", "Java version", contentView),
                 addTogglePref(BuildSettings.SETTING_NO_WARNINGS, true, "Hide warnings in error log", 12, contentView),
                 addTogglePref(BuildSettings.SETTING_NO_HTTP_LEGACY, false, "Don't include http-legacy-28.dex", 12, contentView),
-                addTogglePref(BuildSettings.SETTING_ENABLE_LOGCAT, true, "Enable debug logcat logs viewable in Logcat Reader. Not enabled in exported AABs/APKs.", 12, contentView)
+                // Most of the time, the user doesn't check logcat, so it's disabled by default.
+                addTogglePref(BuildSettings.SETTING_ENABLE_LOGCAT, false, "Enable debug logcat logs viewable in Logcat Reader. Not enabled in exported AABs/APKs.", 12, contentView)
```

Comment:
could you please stop making more changes than just adding r8

Reply:
if you *really* wanna make that change, at least create a separate commit for it

Reply:
and if you're already at that, put that comment into the commit message instead of into the file; you don't need to explain your change so that it's easily visible in the file

Reply:
also, if settings defaults are changed, make sure to sync the new default with the repo, so in this case `a.a.a.yq#a(Context)`